src/js/components/agency/visualizations/objectClass/MajorObjectClasses.jsx

```diff
@@ -245,7 +247,7 @@ export default class MajorObjectClasses extends React.Component {
         return (
             <div className="treemap-inner-wrap">
                 {greatThanOneHundredDescription}
-                { this.createTooltip() }
```

Comment:
There are few places where your code formatter may have removed spaces around only one side of the contents of curly brackets. This isn't throwing a lint error but it would be nice to keep the formatting in tact for consistency.

Reply:
Weird that its rules seem inconsistent...